### PR TITLE
ENH: add doc for SinkhornOutput

### PR DESCRIPTION
**Pull request by pierreablin**
This adds a docstring to the `SinkhornOutput` class, which is often exposed to the user.

### CODE REVIEW

File: src/ott/solvers/linear/sinkhorn.py [hunk 1]
```diff
@@ -279,7 +279,28 @@ def ent_reg_cost(
 
 
 class SinkhornOutput(NamedTuple):
-  """Implements the problems.Transport interface, for a Sinkhorn solution."""
+  r"""Implements the problems.Transport interface, for a Sinkhorn solution.
```
Comment (michalk8) [inline src/ott/solvers/linear/sinkhorn.py:282]: No need for `r`

File: src/ott/solvers/linear/sinkhorn.py [hunk 2]
```diff
@@ -279,7 +279,28 @@ def ent_reg_cost(
...
+
+  Attributes:
+    f: jnp.ndarray, potential
```
Comment (michalk8) [inline src/ott/solvers/linear/sinkhorn.py:285]: Please remove all type hints from docstrings.

File: src/ott/solvers/linear/sinkhorn.py [hunk 3]
```diff
@@ -279,7 +279,28 @@ def ent_reg_cost(
...
+    geom: geometry.Geometry, the geometry used in the problem
+    a: jnp.ndarray, marginals of the problem
+    b: jnp.ndarray, marginals of the problem
+    linear_output: bool, True
+    converged: bool, whether the Sinkhorn solver has converged
+    n_iters: int, the number of iterations of the solver
+    scalings: tuple of jnp.ndarray, the scaling vectors
+    matrix: jnp.ndarray, the transport plan matrix
+    transport_mass: float, sum of transport matrix.
+
```
Comment (michalk8) [inline src/ott/solvers/linear/sinkhorn.py:302]: Remove empty line.

File: src/ott/solvers/linear/sinkhorn.py [hunk 4]
```diff
@@ -279,7 +279,28 @@ def ent_reg_cost(
...
+    f: jnp.ndarray, potential
+    g: jnp.ndarray, potential
+    errors: jnp.ndarray, the list of errors of the Sinkhorn iterations
+    reg_ot_cost: float, the final cost value in the primal space
+    ot_prob: linear_problem.LinearProblem, the problem Sinkhorn was fit on
+    dual_cost: float, the final unregularized cost value in the dual space
+    primal_cost: float, the final unregularized cost value in the primal space
+    linear: bool, whether the problem is linear
+    geom: geometry.Geometry, the geometry used in the problem
+    a: jnp.ndarray, marginals of the problem
```
Comment (michalk8) [inline src/ott/solvers/linear/sinkhorn.py:294]: Source marginals

File: src/ott/solvers/linear/sinkhorn.py [hunk 5]
```diff
@@ -279,7 +279,28 @@ def ent_reg_cost(
...
+    b: jnp.ndarray, marginals of the problem
```
Comment (michalk8) [inline src/ott/solvers/linear/sinkhorn.py:295]: Target marginals

**Review by michalk8 — CHANGES_REQUESTED** [review]
Thanks Pierre! I think it would be best however to work on this in parallel in #284, since some attributes will be removed + documenting dataclasses is a bit different - wdyt?

**Comment by marcocuturi** [comment]
@pierreablin could you take another look, in order to merge/close this?

**Comment by pierreablin** [comment]
Actually seems like https://github.com/ott-jax/ott/commit/db9f58a13755901e877ececa6d2388dd9d62d6b1 makes this useless, right? If yes, we can close.

**Comment by michalk8** [comment]
Closing because of inactivity

**Pull request closed without merging**